src/state/swap/hooks.tsx

```diff
@@ -87,6 +87,8 @@ export function useDerivedSwapInfo(): {
   }
   allowedSlippage: Percent
 } {
+  return undefined as any
+  /* Mod: this hooks is not used
```

Comment:
Why not comment out the whole hook, then?

Reply:
I think because its used in original Uni components and if removed it will require to modify a bunch of other connected code.

Reply:
@nenadV91 is right